src/main/java/dk/sdu/se_f22/contentmodule/index/DB/RecreateTables.java

```diff
@@ -0,0 +1,44 @@
+package dk.sdu.se_f22.contentmodule.index.DB;
+
+public class RecreateTables {
```

Comment:
Maybe the migration process should be used instead of statically re-creating your tables each time. 
This also means it loses it's persistence.